build.gradle

```diff
@@ -40,15 +40,6 @@ buildscript {
     }
 }
 
-//Remove doclint warnings that pollute javadoc logs when building with java8
-if(JavaVersion.current().isJava8()){
```

Comment:
JDK11 failed to execute task `javadoc` (c.f. https://github.com/apache/eventmesh/actions/runs/7406186071/job/20162982838), because of the JavaDoc I wrote in https://github.com/apache/eventmesh/pull/4698/commits/7c1c2ddb72a47fb30ffe4a98a4ac88e69a283088.

I had a re-check and it seems that I have to and another code tag to make JDK11 javadoc build success:

![image](https://github.com/apache/eventmesh/assets/41445332/88ce84c4-bcba-4588-ac77-cd409b85ecb1)

During my troubleshooting process, I noticed that the `javadoc.options` file does not provide any useful information, which led me to test the reason for needing a closing `</p>` tag or indentation or other issues. It was quite time-consuming.

[javadoc.options.txt](https://github.com/apache/eventmesh/files/13848546/javadoc.options.txt)

In your opinion, do you think it would be better to keep the javadoc task settings as they are, only effective for JDK 8, or would it be better to remove the "if" condition for JDK 8?

Reply:
For now, the `javadoc` task remains original. Only executed under JDK8 will surpress `javadoc` warnings.